Dockerfile

```diff
@@ -1,25 +1,19 @@
-# docker build -t vis .
-# docker run -it --name vis vis
-# docker cp . vis:/tmp/vis
-# ./configure CC='cc --static'
-# make
-# docker cp vis:/tmp/vis/vis .
-# make vis-single
-# docker cp vis:/tmp/vis/vis-single .
+# Run 'make docker' to build a statically linked vis executable!
 FROM i386/alpine:3.11
-ENV DIR /tmp/vis
-WORKDIR $DIR
-RUN apk update && apk add musl-dev fortify-headers gcc make libtermkey-dev \
+RUN apk update
```

Comment:
Is there a specific reason you want to split this into two commands? The [Docker documentation](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run) recommends a single `RUN` directive to avoid caching issues.

Reply:
Docker creates read-only layers for every step. Removing caches in a later step does not save the disk space as files are still available in earlier layer. Actually we do not remove any cache. This makes sense only for something like:
```
RUN apk update && apk add musl-dev ... && find /var/cache/apk/ -name 'APKINDEX.*' -delete
```
Looks like `apk` does not cache any package files, so nothing to clean there.

Reply:
The point they are making is: if you add a new package to install, docker will not run the update command, but instead use the cached layer. As a result you might end up with an outdated package.

Reply:
Ah, got it...
Probably less likely to hit as we run with a defined version, but still valid.